prebid-mobile/modules/rendering/android-sdk-integration-pb.md

```diff
@@ -205,29 +249,45 @@ rewardedAdUnit?.show()
 ```
 
 {% capture warning_note %}
-Pay attention that the `loadAd()` should be called on the main thread.
+The `loadAd()` method should be called on the main thread.
 {% endcapture %}
 {% include /alerts/alert_warning.html content=warning_note %}
 
 #### Step 1: Create a Rewarded Ad Unit
 {:.no_toc}
 
-Create the `RewardedAdUnit` object with parameters:
+Create the `RewardedAdUnit` object with the parameters:
 
-* `adUnitId` - an ID of Stored Impression on the Prebid server.
+* `adUnitId` - this is the value of the adunit name in the ad server. e.g. for GAM, it's the adunit name. Other ad servers may call this something different, but the idea is that the buyers want to know which slot in the app is up for auction for targeting and reporting. An example would be "/home/upper-mobile-banner".
 
 #### Step 2: Load the Ad
 {:.no_toc}
 
-Call the `loadAd()` to make a bid request.
+Call the `loadAd()` method to make a bid request.
 
 #### Step 3: Show the Ad when it is ready
 {:.no_toc}
 
-Wait until the ad is loaded and present it to the user in any suitable time.
+Wait until the ad is loaded, then present it to the user. 
 
 ``` kotlin
-override fun onAdLoaded(rewardedAdUnit: RewardedAdUnit) {
-//Ad is ready for display
-}
+// Make an ad request 
+    RewardedAd.load(activity, adUnitId, request, object : RewardedAdLoadCallback() {
```

Comment:
It's irrelevant. Most probably copied from the GMA SDK page, but Rendering API works another way. There is no `RewardedAd.load` here.